Dockerfile

```diff
@@ -1,33 +1,33 @@
 FROM golang:1.19-alpine AS build
-ARG GOTESTWAF_VERSION="unknown"
+RUN apk --no-cache add git
```

Comment:
Dockerfile is not consistent now; you have this block below:
```dockerfile
FROM alpine
WORKDIR /app
RUN apk add --no-cache chromium fontconfig
```
And this block here:
```dockerfile
FROM golang:1.19-alpine AS build
RUN apk --no-cache add git
WORKDIR /app/
```

Can you do the same order in both cases? `FROM`, `RUN`, `WORKDIR` or `FROM`, `WORKDIR`, `RUN`, or maybe in your own way